src/phone_number_input.rs

```diff
@@ -293,6 +286,21 @@ impl PhoneNumberInput {
             .map(|item| item.downcast::<CountryInfo>().unwrap())
     }
 
+    fn set_selected_country_code(&self, position: Option<u32>) {
+        let imp = self.imp();
+        match position {
+            Some(list_pos) => {
+                imp.combo_row.set_selected(list_pos);
+                imp.combo_row.set_subtitle("");
+            }
+            None => {
+                imp.combo_row.set_selected(gtk::INVALID_LIST_POSITION);
+                imp.combo_row
+                    .set_subtitle(&gettext("You entered an invalid country code."));
+            }
+        }
```

Comment:
```suggestion
        let combo_row = &self.imp().combo_row;

        combo_row.set_selected(position.unwrap_or(gtk::INVALID_LIST_POSITION));

        if position.is_some() {
            combo_row.set_subtitle("");
        } else {
            combo_row.set_subtitle(&gettext("You entered an invalid country code."));
        }
```
Not sure about that, possibly also just `if let`

Reply:
Thanks. Yes, this is shorter.